qdrant-landing/themes/qdrant-2024/assets/css/mixins/_mixins.scss

```diff
@@ -10,7 +10,6 @@
   }
   figcaption,
   figcaption p {
-    color: $caption-color;
```

Comment:
<img width="762" alt="Screenshot 2024-10-25 at 4 29 37 PM" src="https://github.com/user-attachments/assets/f77aafef-3fc7-4a82-943f-ee7536d56ada">

Bad visibility in light mode otherwise.